code/game/machinery/autolathe_datums_vr.dm

```diff
@@ -1,20 +1,16 @@
-/datum/autolathe/recipe/classic_smg_9mm
```

Comment:
Since autolathe_datums was removed, this file, autolathe_datums_vr, should be changed to autolathe_vr.

The _vr versions are always "vorestation" mods of the originals, either adding or changing definitions to avoid compatibility issues where possible by simply overriding the original class definition or procs.